packages/modern-poalim-scraper/src/scrapers/types/cal/get-card-transactions-details.ts

```diff
@@ -0,0 +1,155 @@
+export interface CalGetCardTransactionsDetailsResponse {
+  result: Result
```

Comment:
this should be generated from a JSON schema file and also validate it when we get a response
Please take a look at how we do it in the other scrapers

Reply:
Shared a demo with zod in private. Can quickly switch this over to json-to-ts / ajv if preferred